bw2io/importers/ecospold2_biosphere.py

```diff
@@ -19,9 +20,39 @@
 
 
 class Ecospold2BiosphereImporter(LCIImporter):
+    '''Import elementary flows from ecoinvent xml format.
+
+    Parameters
+    ----------
+    name : str, optional
+        Name of the database, by default "biosphere3".
+    version : str, optional
+        Version of the database, by default "3.9".
+
+    Attributes
+    ----------
+    format : str
+        Format of the data: "Ecoinvent XML".
+    db_name : str
+        Name of the database.
+    data : list
+        Extracted data from the xml file.
+    strategies : list
+        List of functions to apply to the extracted data.
```

Comment:
This would be an instance where we should use the 'see also' section of the numpy standard. Link the relevant strategies below: https://github.com/brightway-lca/brightway2-io/tree/main/bw2io/strategies

Reply:
Fixed with https://github.com/brightway-lca/brightway2-io/pull/166/commits/136c40ca7ca064fccadfcb2d50b44788d27ec505